### ISSUE TITLE
Issue with generating UUIDs from BrukerFiles

### ISSUE THREAD

**Issue by tknopp** [MagneticParticleImaging/MPIFiles.jl#32]
From Julia 1.5 to 1.6 something changed such that under Julia 1.5 we have:
```
julia> uuid4(MersenneTwister(1))
UUID("5b7e9947-ddc0-4b3f-9b55-0d8042f74170")
```
and under 1.6
```
julia> uuid4(MersenneTwister(1))
UUID("a6e29944-5f81-4247-9979-0d8042f74170")
```
This is not fully surprising when reading the warning on this page:
https://docs.julialang.org/en/v1/stdlib/Random/

The issue now is that for existing BrukerFiles completely new UUIDs are generated, due to this code:
https://github.com/MagneticParticleImaging/MPIFiles.jl/blob/master/src/Brukerfile.jl#L186

What should we do here? I fear that there is not really a possibility of getting the 1.5 behavior in 1.6.  Ping @hofmannmartin

**Comment by tknopp** [comment]
Solution 1: 

use https://github.com/JuliaRandom/StableRNGs.jl

Solution 2: 

try to derive the UUID directly from Brukers string. For instance one could pick individual letters. I think we once did this.

**Comment by hofmannmartin** [comment]
I would vote for solution 1:

**Comment by jusack** [comment]
Since we had the same situation again going from 1.10 to 1.11 we have implemented solution 1 in 997cf728c0790b6fa71141957cc30d349259e385
